### PR TITLE
Use skopeo to push image as workaround for "unknown blob"

### PR DESCRIPTION
**Pull request by dhiller**
As `docker push` is failing nearly everytime we use skopeo to push images into the registry.

Local run succeeded in pushing images with tag `2202141654-55f845f3`

### CODE REVIEW

File: publish.sh [hunk 1]
```diff
@@ -25,7 +26,11 @@ done
 IMAGES="${CLUSTERS}"
 docker push ${TARGET_REPO}/gocli:${KUBEVIRTCI_TAG}
```
Comment (brianmcarey) [inline publish.sh:27]: @dhiller is a skopeo copy required here too?

Comment (dhiller) [reply publish.sh:27]: Good point :blush:

Comment (dhiller) [reply publish.sh]: @brianmcarey fixed, PTAL, thanks!

**Comment by enp0s3** [comment]
/approve
/hold
feel free to unhold upon CI pass

**Pull request merged**